Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -71,11 +65,12 @@ class NotificationsViewController: UIViewController {
             switch response {
             case .success(let response):
                 guard let strongSelf = self else { return }
-                strongSelf.pendingRequests = response.waitingApproval
-                strongSelf.respondedRequests = response.toApprove
-                strongSelf.allRequests = strongSelf.pendingRequests + strongSelf.respondedRequests
+                strongSelf.pendingRequests = response.pendingRequests
+                strongSelf.awaitingApproval = response.awaitingApproval
+                strongSelf.allRequests = strongSelf.pendingRequests + strongSelf.awaitingApproval
+                strongSelf.setupTableView()
```

Comment:
Personally I use `guard let self = self else { return }` because `strongSelf` is very long but up to you.

Reply:
hmm yeah this was w. reade's comments from my last PR

Reply:
Yea the unwrapped version is the strong reference so some people like to call it strongSelf but it's just a matter of preference; it does the same thing.